# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/hwdata/no-pci-usb-ids.patch
# Copyright (C) 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

From: "VS" <vs@salvatorenco.fr>
Date: Mon, 14 Nov 2022 14:34 +0100
Subject: [PATCH] Update wlroots and related packages

Don't install pci.ids nor usb.ids as they are respectlively
supplied with pciutils and usbutils.
--- hwdata-0.364/Makefile.orig	2022-11-14 13:03:05.498727784 +0100
+++ hwdata-0.364/Makefile	2022-11-14 13:05:04.189742319 +0100
@@ -14,10 +14,10 @@
 
 CVSTAG = $(NAME)-r$(subst .,-,$(VERSION))
 
-IDFILES = pci.ids usb.ids oui.txt iab.txt pnp.ids
+IDFILES = oui.txt iab.txt pnp.ids
 
 # usb.ids is not in UTF-8
-UTF_IDFILES = pci.ids oui.txt iab.txt pnp.ids
+UTF_IDFILES = oui.txt iab.txt pnp.ids
 
 .PHONY: all install tag force-tag check commit create-archive archive srpm-x \
     clean clog download
